### PR TITLE
Add Argon

### PR DESCRIPTION
**Pull request by diedyesterdaywastaken**
### SteamDB app page links to a few games using this

- [Chernobyl Commando](https://steamdb.info/app/302240/)
- [Chernobyl: Terrorist Attack](https://steamdb.info/app/546810/)
- [Farm Expert 2016](https://steamdb.info/app/334750/)
- [Farm Expert 2017](https://steamdb.info/app/424590/)
- [Farm Machines Championships 2014](https://steamdb.info/app/284290/)
- [Farmer's Dynasty](https://steamdb.info/app/678900/)
- [Lumberjack's Dynasty](https://steamdb.info/app/968970/)
- [Manhunter](https://steamdb.info/app/302250/)
- [Redneck Racers](https://steamdb.info/app/288730/)
- [Trucker 2](https://steamdb.info/app/392890/)

### CODE REVIEW

**Comment by SteamDB-Tracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.Argon**

These apps now match: **(16)** [258880](https://steamdb.info/app/258880/depots/) [264930](https://steamdb.info/app/264930/depots/) [284290](https://steamdb.info/app/284290/depots/) [288730](https://steamdb.info/app/288730/depots/) [302240](https://steamdb.info/app/302240/depots/) [302250](https://steamdb.info/app/302250/depots/) [316240](https://steamdb.info/app/316240/depots/) [326350](https://steamdb.info/app/326350/depots/) [334750](https://steamdb.info/app/334750/depots/) [392890](https://steamdb.info/app/392890/depots/) [399940](https://steamdb.info/app/399940/depots/) [424590](https://steamdb.info/app/424590/depots/) [546810](https://steamdb.info/app/546810/depots/) [678900](https://steamdb.info/app/678900/depots/) [968970](https://steamdb.info/app/968970/depots/) [2913560](https://steamdb.info/app/2913560/depots/)

Please confirm that all these changed apps are correct.

File: descriptions/Engine.Argon.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[**Buddha**](http://www.silden.eu/technology.html) is an in-house engine used to power games made by [Sliden](http://www.silden.eu/).
```
Comment (Lifeismana) [inline descriptions/Engine.Argon.md:1]:
```suggestion
[**Argon**](http://www.silden.eu/technology.html) is an in-house engine used to power games made by Sliden.
```

File: rules.ini [hunk 2]
```diff
@@ -40,6 +40,7 @@ AGI = (?:^|/)AGIDATA\.OVL$
 Alpha_Engine = ^Alpha_Engine\.dll$
 Amazon_Lumberyard = ^bootstrap\.cfg$
 AppGameKit = (?:^|/)bytecode\.byc$
+Argon = (?:^|/)ar\.cfg$
```
Comment (Lifeismana) [inline rules.ini:43]:
```suggestion
Argon = ^ar\.cfg$
```

**Review by Lifeismana — CHANGES_REQUESTED** [review]
needs non matching tests

File: descriptions/Engine.Argon.md (hunk 1 above)
Comment (diedyesterdaywastaken) [reply descriptions/Engine.Argon.md]: didnt hit ctrl+s, my fault

**Pull request merged**